### PR TITLE
Bug/fix error variants

### PR DESCRIPTION
**Pull request by marksy**
- fixed components that had the error text not working
- updated snapshots
- fixed test
- changed false ternary values to `undefined`

### CODE REVIEW

File: src/components/Button/index.js [hunk 1]
```diff
@@ -39,16 +39,16 @@ const Button = glamorous.button(
...
-    fontSize: start ? "24px" : Button.fontSize,
-    lineHeight: start ? "1.25" : Button.lineHeight,
-    padding: start ? "0.36842em 0.84211em 0.21053em" : Button.padding
+    fontWeight: start ? "700" : undefined,
+    fontSize: start ? "24px" : undefined,
+    lineHeight: start ? "1.25" : undefined,
+    padding: start ? "0.36842em 0.84211em 0.21053em" : undefined
   }),
   ({ iconUrl }) => ({
     backgroundImage: iconUrl ? `url( ${iconUrl})` : "",
```
Comment (penx) [inline src/components/Button/index.js]: should this be `: "none",`?

Comment (penx) [reply src/components/Button/index.js]: or undefined to not set it at all

File: src/components/Select/__snapshots__/test.js.snap [hunk 2]
```diff
@@ -193,9 +193,6 @@ exports[`Select matches snapshot: enzyme.mount 1`] = `
   flex-direction: column;
   box-sizing: border-box;
   width: 100%;
-  border-left: ;
```
Comment (penx) [inline src/components/Select/__snapshots__/test.js.snap]: I wonder if there's a way we can have a unit test that validates that css output validates

**Comment by penx** [comment]
Can you add tests that make sure that, if this happens again, the tests will pick it up?

File: src/components/Select/__snapshots__/test.js.snap (hunk 2 above)
Comment (marksy) [reply src/components/Select/__snapshots__/test.js.snap]: would be good to get css linting working, but the plugins for `css-in-js`/glamorous aren't mature enough/working as expected

**Pull request merged**